services/metadata/backend/data/lenzburg.json

```diff
@@ -17,7 +17,7 @@
+            "en": "The Folkfestival auf der Lenzburg (Lenzburg Folk Festival) took place in Switzerland from 1972 until 1980 at Lenzburg Castle (Lenzburg, Aargau), always on a weekend in June or July. Looking back the festival was the most important folk event in Switzerland. Over the years, many Swiss and international folk bands played at the Lenzburg festival. The event grew continuously: its duration was extended from initially one to two days, more international artists from different subgenres of folk performed, and also the audience grew steadily so that the castle became gradually very crowded, and tickets had to be sold by drawing lots. For this reason, and because various political and social upheavals were shaking up the Swiss folk scene (furthermore, reconstruction works on the castle were planned), many members of the organizing committee resigned, and the last festival was held in 1980. \nThe festival was mainly the result of the efforts of musicians and other actors around the folk clubs of Bern and Zurich which formed an association and an organizing committee. The Lenzburg Folk Festival Society made tape recordings on cassette and magnetic tapes during the performances. In 1984, the audio tapes were donated to the Schweizerisches Volksliedarchiv (Swiss Folk Song Archive, SVA) in Basel, accompanied by a typewritten documentation, archival chards, and further archival materials, e.g., program leaflets (documentation process continued until 1987). \nThe Lenzburg audio tape collection was archived, stored, and for a long time little was done with the tapes. It was not until 2015 that the SVA decided to breathe new life into the recordings and digitize them. Two reasons were decisive for this: firstly, the SVA as a section of the archives of the Swiss Folklore Society aims at making the rich historical folk culture collections accessible to the broader public and for scientific use. Secondly, practical reasons create a demand for digitization since the SVA cannot create the optimal conditions for storing audio tapes, and the archive does not own the technical devices for magnetic audio tapes so that the recordings could therefore no longer be listened to by the public.\nThe Lenzburg project started in 2017 and was presented to the public for the first time at an event of the Cultural Anthropology Switzerland (CAS) section Basel, on February 11, 2019, in the form of a workshop report."
```

Comment:
you forgot to change 1 occurrence of "Swiss Folklore Society" (The line is so long that I can't give you a line number, but with search-replace you will find it :-)